.github/workflows/ci.yml

```diff
@@ -138,9 +139,14 @@ jobs:
           - os: windows
             ruby: "3.1"
             platform: x64-mingw-ucrt
+          - os: windows
+            ruby: "3.2"
```

Comment:
Sorry, RubyInstaller-3.2.0 is not yet released. I'll probably release it today.

Reply:
I'm changing this PR to use "head" so it's not a blocker!

Reply:
But head is already ruby-3.3 and we don't support cross builds for development versions, do we?

Reply:
Well, look, I'm just trying to get to green. Tell me what you'd like me to do.

Reply:
You're right, of course. I'll leave it failing for now.